symfony/plugins/orangehrmRESTPlugin/lib/Api/Pim/Entity/Employee.php

```diff
@@ -53,6 +53,8 @@ class Employee implements Serializable
 
     private $workEmail;
 
+    private $otherEmail;
```

Comment:
any reason to change this file?

Reply:
it was done on mistake, updated the correct Employee entity file